### PR TITLE
EZP-26869: Documented in-context translation usage

### PR DESCRIPTION
**Pull request by bdunogier**
Title says it all.

### CODE REVIEW

File: doc/i18n/in_context_translation.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+### In-context UI translation
+
+Since eZ Platform 1.7.0, the interface has been fully translatable. Version 1.8.0 introduces
+official support for [crowdin.com](crowdin.com) as a translation management system. In
+addition, it integrates support for [in-context translation](in-context), a feature that
+allows you to translate strings from the interface, _in context_.
+
+![In-context translation of Platform UI](https://cloud.githubusercontent.com/assets/235928/21649816/44fc2ea0-d2a3-11e6-8c0e-1b5493ea47e9.png)
```
Comment (yannickroger) [inline doc/i18n/in_context_translation.md]: Should we store this screenshot along with the documentation?

File: doc/i18n/in_context_translation.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+The first time you enable in-context, if you're not logged into Crowdin, it will ask you
+to login or register an account. Once done, it will ask you which language you want to
+translate in, from the list of configured languages in crowdin.
+
+Choose your language, and you can start translating right away. Translatable strings in the
+interface will be outlined in red (untranslated), blue (translated) or green (approved).
+When moving over them, an edit button will show up on the top left corner of the outline.
+Click on it, and edit the string in the window that shows up.
+
+## Troubleshooting
```
Comment (yannickroger) [inline doc/i18n/in_context_translation.md]: You should add another paragraph explaining what is going to happen next. Like when the translations will actually be integrated in the product.

File: doc/i18n/in_context_translation.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+### In-context UI translation
+
+Since eZ Platform 1.7.0, the interface has been fully translatable. Version 1.8.0 introduces
+official support for [crowdin.com](crowdin.com) as a translation management system. In
+addition, it integrates support for [in-context translation](in-context), a feature that
+allows you to translate strings from the interface, _in context_.
+
+![In-context translation of Platform UI](https://cloud.githubusercontent.com/assets/235928/21649816/44fc2ea0-d2a3-11e6-8c0e-1b5493ea47e9.png)
+
+## Toggling in-context translation
```
Comment (yannickroger) [inline doc/i18n/in_context_translation.md]: How does it work regarding the version used by the user. I mean 1.8.x or master etc.

File: doc/i18n/in_context_translation.md (hunk 1 above)
Comment (bdunogier) [inline doc/i18n/in_context_translation.md]: Maybe, maybe not. @mere-teresa should confirm that, but storing images is one of the open questions when it comes to doc & github. Based on what I have seen, using images stored via pull-requests / issues is a viable way to do it, and it is simple. We can change it at some point indeed.

File: doc/i18n/in_context_translation.md (hunk 3 above)
Comment (bdunogier) [inline doc/i18n/in_context_translation.md]: Will work starting from the hash where this was merged to ezplatform. So from master of yesterday and 1.8.0.

File: doc/i18n/in_context_translation.md (hunk 2 above)
Comment (bdunogier) [inline doc/i18n/in_context_translation.md]: I'm not yet sure about that :)

Comment (SylvainGuittard) [inline doc/i18n/in_context_translation.md]: This documentation page is not related to the translation in the project. It's only "how to use in-context-translation". I would not add the "next steps".

File: doc/i18n/in_context_translation.md (hunk 1 above)
Comment (yannickroger) [inline doc/i18n/in_context_translation.md]: We should at some point as it kills the value of being able to read the documentation offline.

File: doc/i18n/in_context_translation.md (hunk 3 above)
Comment (yannickroger) [inline doc/i18n/in_context_translation.md]: So the fact that the doc is there means the version is able to handle that :)
